.github/workflows/debian.yml

```diff
@@ -21,19 +21,20 @@ on:
       - "**.md"
   workflow_dispatch:
 
+env:
+  IKARUS_PYTHON_TEST_BUILD_TYPE_OVERRIDE: "Release"
```

Comment:
Is this really recognized? Did you check? Maybe we can move this to the matrix and just write before the cmake lines `export IKARUS_PYTHON_TEST_BUILD_TYPE_OVERRIDE=${{ matrix.config.config }}` Then we can later easily add a Python debug config.

Reply:
Yes this is recognized, but this is a good idea to set it like that, Ill try an Python debug config, I want to see what is faster actually to run through